serverless/event_sources/serverless-apiserversource.adoc

```diff
@@ -1,13 +1,13 @@
 include::modules/serverless-document-attributes.adoc[]
 [id="serverless-apiserversource"]
-= Using ApiServerSource
+= Using the API server source
 include::modules/common-attributes.adoc[]
 :context: serverless-apiserversource
 
 toc::[]
 
-ApiServerSource is an event source that can be used to connect an event sink, such as a Knative service, to the Kubernetes API server.
-ApiServerSource watches for Kubernetes events and forwards them to the Knative Eventing broker.
+An API server source is an event source that can be used to connect an event sink, such as a Knative service, to the Kubernetes API server.
+An API server source watches for Kubernetes events and forwards them to the Knative Eventing broker.
```

Comment:
Earlier, you had "Broker" capitalized, so pointing this out for consistency.

Reply:
Should be lowercase everywhere so I'll check that, thanks

Reply:
I did a quick grep on this and I don't think it's capitalized now anywhere except in the YAML examples where it's referring to a specific API object type. Let me know if there are any specific instances you see that need to change please :)

Reply:
SGTM, thanks!